app/src/main/java/org/oppia/android/app/settings/profile/ProfileResetPinViewModel.kt

```diff
@@ -8,10 +8,21 @@ import javax.inject.Inject
 /** The ViewModel for [ProfileResetPinActivity]. */
 @ActivityScope
 class ProfileResetPinViewModel @Inject constructor() : ObservableViewModel() {
+  /** variable for storing error message in first text input layout. */
   val pinErrorMsg = ObservableField("")
+
+  /** variable for storing error message in second text input layout. */
   val confirmErrorMsg = ObservableField("")
+
+  /** for confirming if the profile is admin or not. */
   val isAdmin = ObservableField(false)
+
+  /** for storing input pin entered by user in first text input layout. */
   val inputPin = ObservableField("")
+
+  /** for storing input pin entered by user in second text input layout. */
```

Comment:
First character capital in all comments.

Reply:
done